src/h3lib/lib/polyfill.c

```diff
@@ -700,6 +706,17 @@ static double getAverageCellArea(int res) {
 H3Error H3_EXPORT(maxPolygonToCellsSizeExperimental)(const GeoPolygon *polygon,
                                                      int res, uint32_t flags,
                                                      int64_t *out) {
+    // Special case: 0-vertex polygon
+    if (polygon->geoloop.numVerts == 0) {
+        *out = 0;
+        return E_SUCCESS;
+    }
+    // Special case: 1-vertex polygon
+    if (polygon->geoloop.numVerts == 1) {
+        *out = 1;
+        return E_SUCCESS;
+    }
```

Comment:
In overlapping BBOX mode, this will not be correct as more than one cell bounding box may overlap a vertex.

Reply:
Gotcha. I'll have a new PR up for this soon.